lib/iris/coords.py

```diff
@@ -1233,6 +1233,11 @@ def _guess_bounds(self, bound_position=0.5):
             - the units are degrees,
             - all the points are in the range [-90, 90].
 
+        .. deprecated:: 2.0.0
+            The 'iris.FUTURE.clip_latitudes' option is now deprecated
+            and is set to True by default. Please remove code which
+            uses 'iris.FUTURE.clip_latitudes'.
```

Comment:
This deprecation warning needs to focus on the behaviour change in this method, not on the FUTURE flag.

The reference to the FUTURE needs to be surrounded with backticks \` and not apostrophes.